builds/ob-release-notes.adoc

```diff
@@ -0,0 +1,31 @@
+:_content-type: ASSEMBLY
+[id="ob-release-notes"]
+= Release notes
+include::_attributes/common-attributes.adoc[]
+:context: ob-release-notes
+
+toc::[]
+
+Release notes contain information about new and deprecated features, breaking changes, and known issues. The following release notes apply for the most recent {builds-shortname} releases on {ocp-product-title}.
+
+{builds-title} is an extensible build framework based on the link:https://shipwright.io/[Shipwright project], which you can use to build container images on the {ocp-product-title} cluster. You can build container images from source code and Dockerfile using image build tools, such as Source-to-Image (S2I) and Buildah. You can create and apply build resources, view logs of build runs, and manage builds in your {ocp-product-title} namespaces.
+
+{builds-title} provides:
```

Comment:
This lead in sentence is incomplete. How about:

{builds-title} includes the following capabilities: